3rdparty/vendor/onelogin/php-saml/README.md

```diff
@@ -1169,7 +1169,7 @@ Main class of OneLogin PHP Toolkit
  * `getNameId` - Returns the nameID
  * `getNameIdFormat` - Gets the NameID Format provided by the SAML response from the IdP.
  * `getNameIdNameQualifier` - Gets the NameID NameQualifier provided from the SAML Response String.
- * `getNameIdNameSPQualifier` - Gets the NameID SP NameQualifier provided from the SAML Response String.
```

Comment:
Could you remove the changes from 3rdparty/ ? Documentation is already fixed upstream with https://github.com/onelogin/php-saml/commit/ead86cf9ac7d2218a470aff029782db3eac13032 and we get that once we update the vendor package again.